config/pharmacies/transform.js

```diff
@@ -0,0 +1,68 @@
+const days = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday'];
+const minutesInADay = 1440;
+
+function flattenArray(arrayOfArrays) {
+  return arrayOfArrays.reduce((a, b) => a.concat(b), []);
+}
+
+function getMinutesOffset(dayCount, time) {
+  return (minutesInADay * dayCount) +
+    (Number(time.substring(0, 2)) * 60) +
```

Comment:
Would splitting `time` be easier to read than the 2 `substring`s? It would also prevent issues with there not being a leading 0 for single hour times. e.g.
```js
const timeSplit = time.split(':')
Number(timeSplit[0] * 60) + 
Number(timeSplit[1] * 60)
```